src/account/Settings.test.tsx

```diff
@@ -470,7 +470,7 @@ describe('Account', () => {
       body: '{"phoneNumber":"+14155550000","clientPlatform":"android","clientVersion":"0.0.1"}',
     })
     expect(navigate).toHaveBeenLastCalledWith(Screens.BackupPhrase, {
-      settingsScreen: Screens.SettingsDrawer,
+      settingsScreen: Screens.Settings,
```

Comment:
can we keep one test for SettingsDrawer and another for Settings?

Reply:
I actually meant keeping one of the existing delete tests for drawer and one for non drawer like how it was prior to the original change. PN being verified / unverified is independent of this screen. We don't need all combinations, but also doesn't hurt except for a tiny bump in CI time.